core/packages/PackageDB.cc

```diff
@@ -217,6 +217,10 @@ const std::vector<core::NameRef> &PackageDB::layers() const {
     return layers_;
 }
 
+const bool PackageDB::layeringChecksEnabled() const {
```

Comment:
```suggestion
const bool PackageDB::enforceLayering() const {
```

to match the name of the lint rules?

completely stylistic, feel free to ignore.